containers/gds/Dockerfile

```diff
@@ -25,7 +25,7 @@ COPY pkg ./pkg
 COPY proto ./proto
 
 # Run tests to avoid building a broken container
-RUN go test -timeout 30s -v ./...
+RUN go test -timeout 180s ./...
```

Comment:
Assuming we bumped this up because we were observing premature timeouts?

Reply:
I only saw it happen once, and I don't think it was related to our tests, it was related to the Actions environment; however our tests will only become more comprehensive so I figured it wouldn't hurt to bump the timeout.